docs/styles/scrollbar_gutter.md

```diff
@@ -0,0 +1,34 @@
+# Scrollbar gutter
```

Comment:
Could you add a Syntax section. See the docs-intro branch for examples...

Reply:
Added `Syntax` section, with a child `Values` section.

The `Values` section is probably where we'd link off to "types" e.g. color, scalar, etc.

<img width="726" alt="image" src="https://user-images.githubusercontent.com/5740731/185596644-fd9da9e2-700d-4b07-9af2-8951778daedd.png">

Reply:
I like the Values section. Will add a task to review existing styles.